libraries/AP_Scripting/applets/runcam_on_arm.lua

```diff
@@ -57,8 +55,6 @@ function update()
          rc:run_aux_function(RC_OPTION.RunCamControl, AuxSwitchPos.LOW)
       end
       delay = DELAY_LONG
-   else
-      delay = DELAY_SHORT
```

Comment:
libraries/AP_Scripting/applets/runcam_on_arm.lua:47:10: value assigned to variable delay is unused